certs/readme.md

```diff
@@ -33,3 +33,34 @@ A couple of examples:
 ## Additional Information
 
 For additional guidance and important information about certificates, please refer to [this blog post](https://techcommunity.microsoft.com/t5/internet-of-things/azure-iot-tls-critical-changes-are-almost-here-and-why-you/ba-p/2393169) from the security team.
+
```

Comment:
nit: do we need bearssl?

Reply:
Good point. we won't be supporting that (except if Hub decides to prefer ECC).
1. I can't find an easy way to disable RSA during compilation of BearSSL.
2. We don't allow ciphersuite selection today. We call `br_ssl_client_init_full` which is using all ciphersuites (recommended if size is not an issue).